test/helpers/rpc.js

```diff
@@ -0,0 +1,25 @@
+const pify = require("pify");
+
+/**
+ * Generic RPC interface
+ *
+ * @param {string} method Name of provider method call
+ * @param {Array} params Argument for method call
+ * @param {Object} web3 Web3 interface
+ */
+
+const rpcSend = (method = "", params = [], web3) => {
+  const send = pify(web3._provider.send.bind(web3._provider));
+  // an "id" is required here because the web3 websocket provider (v1.0.0-beta.35) throws if it is
+  // missing (it's probably just a bug on their end)
+  return send({
+    id: "1",
+    jsonrpc: "2.0",
+    method,
+    params
+  });
+};
+
+module.exports = {
+  rpcSend
```

Comment:
This helper could be useful in the future within the forking methods ganache-core in the future. Maybe we should move it to `./lib/utils` now? Thoughts on doing this now vs later?

Also, rename `rpcSend` to just `send` in both spots in this file.

Reply:
Thinking ... we should hold off on updating `./lib/utils` for new tests.

Can definitely swap `rpcSend` to `send`, didn't want it to be confused with existing send().